src/Proto.Actor/EventStream.cs

```diff
@@ -56,9 +56,18 @@ public Subscription<T> Subscribe(Action<T> action)
 
         public void Publish(T msg)
         {
+            var logger = Log.CreateLogger<EventStream>();

```

Comment:
It's not the same thing as the logger in the constructor gets hoisted to a member variable.  I probably should have just made the logger a member variable anyway.